bokehjs/src/coffee/models/glyphs/webgl/line.coffee

```diff
@@ -652,12 +652,13 @@ class LineGLGlyph extends BaseGLGlyph
 
     draw: (indices, mainGlyph, trans) ->
       mainGlGlyph = mainGlyph.glglyph
+      nvertices = mainGlGlyph.nvertices
 
       if mainGlGlyph.data_changed
         if not (isFinite(trans.dx) and isFinite(trans.dy))
           return  # not sure why, but it happens on init sometimes (#4367)
         mainGlGlyph._baked_offset = [trans.dx, trans.dy]  # float32 precision workaround; used in _bake() and below
-        mainGlGlyph._set_data()
+        mainGlGlyph._set_data(nvertices)
```

Comment:
Does this help? I would be surprised, since the `_set_data()` method of this class does not accept any arguments. And I am pretty sure that mainGlGlyph should be an object of the same type...